framework/db/ColumnSchemaBuilder.php

```diff
@@ -21,6 +21,16 @@
 class ColumnSchemaBuilder extends Object
 {
     /**
+     * The following are the categories of column data types.
+     * @since 2.0.8
+     */
+    const CAT_PK = 'pk';
```

Comment:
PhpDoc above will stick to `CAT_PK` in both IDEs and API docs. Regular `//` comment should be used.